### PR TITLE
Det cuts

### PR DESCRIPTION
**Pull request by msilvafe**
Adds in flags function and preprocess support for making detector level cuts based on the IV + bias step results.

### CODE REVIEW

**Comment by msilvafe** [comment]
Ok @kmharrington and @mhasself I have added the changes to change the name from cuts -> flags and wrap a RangesMatrix instead of a boolean mask. I think this is ready for review. Once merged I can run on all observations + create a next context to load in these cuts.

**Review by kmharrington — APPROVED** [review]
My only question is if we want 0.01 or 0.1 to be the minimum acceptable on-transition value. I kind of assume a detector actually at 1% Rn is latched or going to latch very soon.

**Comment by msilvafe** [comment]
> My only question is if we want 0.01 or 0.1 to be the minimum acceptable on-transition value. I kind of assume a detector actually at 1% Rn is latched or going to latch very soon.

I switched it to 0.1 and will merge after build tests.

File: sotodlib/tod_ops/flags.py
Comment (mhasself) [inline sotodlib/tod_ops/flags.py:33]: psat_range is not documented.  Especially important to say the units.

File: sotodlib/tod_ops/flags.py
Comment (mhasself) [inline sotodlib/tod_ops/flags.py:43]: Clarify whether True means cut or uncut.

**Review by mhasself — CHANGES_REQUESTED** [review]
See two doc requests.

**Comment by msilvafe** [comment]
> See two doc requests.

Ok, I've updated those doc requests. Assuming your happy can you approve those changes? Thanks.

**Review by mhasself — APPROVED** [review]

**Pull request merged**
